repos/system_upgrade/el7toel8/actors/checklocalnetworkscripts/libraries/library.py

```diff
@@ -0,0 +1,26 @@
+import os
+from leapp import reporting
+
+
+def generate_report():
+    """ Generate reports informing user about possible manual intervention requirements """
+
+    DOC_URL = "https://red.ht/upgrading-RHEL-8-to-RHEL-9-network-scripts"
+
+    # Show documentation url if local scripts of network-scripts installed
+    title = "local scripts of network-scripts detected"
+    summary = "Upgrade will be stopped. Additional steps might be required.\n\n"
+
+    reporting.create_report(
+        [
+            reporting.Title(title),
+            reporting.Summary(summary),
+            reporting.Severity(reporting.Severity.HIGH),
+            reporting.Flags([reporting.Flags.INHIBITOR]),
+            reporting.Tags([reporting.Tags.SERVICES]),
+            reporting.ExternalLink(
+                title="Please migrate custom network-scripts to NetworkManager dispatcher scripts.",
```

Comment:
Ideally title of the article, not whatever text - it should not be a text "do please this" or "do this", but title of the link.

Reply:
Updated to use the title of the link.